### PR TITLE
hack to remove lit dev mode warning

### PR DESCRIPTION
**Pull request by nielslyngsoe**
Hack to remove lit dev mode warning.

### CODE REVIEW

**Review by iOvergaard — APPROVED** [review]
Amazingly wonderful hack 😍

**Comment by iOvergaard** [comment]
Seems like it doesn't quite work on the test run:
image

File: web-test-runner.config.mjs [hunk 1]
```diff
@@ -140,6 +140,16 @@ export default {
 			</head>
       <body>
         <script type="module" src="${testFramework}"></script>
+				<script type="module">
+					<!-- Hack to disable Lit dev mode warnings -->
```
Comment (iOvergaard) [inline web-test-runner.config.mjs:144]: This needs to be a javascript comment
```suggestion
					// Hack to disable Lit dev mode warnings
```

**Review by iOvergaard — CHANGES_REQUESTED** [review]

**Comment by nielslyngsoe** [comment]
@iOvergaard oh, sorry, my comment ruined the code.. 2sec.

File: web-test-runner.config.mjs (hunk 1 above)
Comment (nielslyngsoe) [reply web-test-runner.config.mjs]: Oh, I saw the problem before your suggestion. So did this already. :-/

**Review by iOvergaard — APPROVED** [review]

**Pull request merged**